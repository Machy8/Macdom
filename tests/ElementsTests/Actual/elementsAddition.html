beer $Gambrinus
	pong $3:1 $Czech team;
