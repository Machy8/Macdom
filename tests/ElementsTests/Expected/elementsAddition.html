<beer brand="Gambrinus">
	<pong score="3:1" winner="Czech team">
</beer>
